notebooks/rxrx3_core_benchmarks_openphenom.ipynb

```diff
@@ -0,0 +1,2858 @@
+{
```

Comment:
<p>Super minor, but this could be its own <code>pcacs</code> function to be imported from <code>efaar_benchmarking/efaar.py</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/recursionpharma/EFAAR_benchmarking/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>